bin/update-ips.bash

```diff
@@ -109,13 +165,53 @@ diffKubectlIPs() {
 
 # Updates the list from the file and yaml path specified with IPs fetched from the nodes
 updateKubectlIPs() {
+    cloud="${1}"
+    label="${2}"
+    inputSource="${3}"
+    inputConfig="${4}"
+
     local IPS
-    read -r -a IPS <<< "$(getKubectlIPs "${1}" "${2}")"
+    read -r -a IPS <<<"$(getKubectlIPs "$cloud" "$label")"
 
-    yq4 -i "${3}"' = []' "${4}"
-    for ip in "${IPS[@]}"; do
-        yq4 -i "${3}"' |= . + ["'"${ip}"'/32"]' "${4}"
-    done
+    processIPRanges "$inputSource" "$inputConfig"
+}
+
+# Process ip ranges with subnet masks in consideration.
+processIPRanges(){
+    local inputSource="$1"
+    local inputConfig="$2"
+
+    local multiIPRanges=()
+    read -r -a multiIPRanges <<<"$(getConfigIPS "$inputSource" "$inputConfig")"
+
+    local working_subnet=()
```

Comment:
Is this actually modified in the other function even if this is a `local` variable?

Reply:
You could say, yes!